xc/common/cmake/install.cmake

```diff
@@ -56,20 +56,20 @@ function(DEFINE_XC_TOOLCHAIN_TARGET)
             ${symbiflow-arch-defs_SOURCE_DIR}/utils/vpr_io_place.py
             ${symbiflow-arch-defs_SOURCE_DIR}/utils/vpr_place_constraints.py
             ${symbiflow-arch-defs_SOURCE_DIR}/utils/eblif.py
-          DESTINATION bin/python
+          DESTINATION bin/python-utils
```

Comment:
Why did we rename this?

Reply:
The idea behind this PR is also to get the install package to be installed via conda.
The `python` directory within the `bin` would generate a conflict with the pre-existing conda `python`. In this way, no conflict is generated. I could as well move all the python scripts within the `share/symbiflow/scripts/` directory though.

Reply:
> I could as well move all the python scripts within the `share/symbiflow/scripts/` directory though.

Ya, please move everything under `share/symbiflow`.